src/openms/source/ANALYSIS/MAPMATCHING/FeatureGroupingAlgorithmKD.cpp

```diff
@@ -66,6 +66,10 @@ namespace OpenMS
     defaults_.setMinFloat("link:rt_tol", 0.0);
     defaults_.setValue("link:mz_tol", 10.0, "m/z tolerance (in ppm or Da)");
     defaults_.setMinFloat("link:mz_tol", 0.0);
+    defaults_.setValue("link:charge_merging","Zero","whether to disallow charge mismatches (None), allow to link charge zero (i.e., unknown charge) with every other charge (Zero), or disregard charges (All).");
+    defaults_.setValidStrings("link:charge_merging", ListUtils::create<String>("None,Zero,All"));
+    defaults_.setValue("link:adduct_merging","All","whether to only allow the same adduct for linking (None), allow linking adduct-free features with other features (Unknowns), or disregard adducts (All).");
```

Comment:
'None' seems to be an unfortunate name for this (implies no merging), but means 'Identical'?

'All' could be 'Any'?